### DISCUSSION TITLE
Testing of port features in CI

### DISCUSSION THREAD

**Discussion by jreichel-nvidia** [microsoft/vcpkg#42602] [Q&A] [answered]
PR #42175 (Update of OpenImageIO to 3.x) broke `mdl-sdk[openimageio]` (fixed in PR #42492). I'm trying to understand why the CI tests for the first PR did not detect this.

My guess is that is that the CI tests only ports without any optional features, i.e., `mdl-sdk[core]` (which does not depend on OpenImageIO)? If yes, is there a simple way to tell the CI to test a port with all its optional features? I guess declaring these features as default features would be one option, but maybe there are different/better ways?

(Edit: fixed typo)

**Comment by Osyotr** [comment, marked as answer]
Features are tested in test ports, for example https://github.com/microsoft/vcpkg/tree/master/scripts/test_ports/vcpkg-ci-gdal

**Reply by jreichel-nvidia** [reply]
Thanks, I'll try that.
